internal/pubsub/mempubsub/mem.go

```diff
@@ -37,8 +52,16 @@ type topic struct {
 
 // OpenTopic establishes a new topic.
 // Open subscribers for the topic before publishing.
-func OpenTopic() driver.Topic {
-	return &topic{}
+func (b *Broker) OpenTopic(ctx context.Context, name string) *pubsub.Topic {
+	b.mu.Lock()
+	defer b.mu.Unlock()
+	t := newTopic(name)
+	b.topics[name] = t
+	return pubsub.NewTopic(ctx, t)
```

Comment:
Side note: Does `pubsub.NewTopic` really need a `ctx`? For blob it doesn't.

Reply:
Yes, it needs a ctx because it spins up a bundler whose handler uses the ctx to make RPCs. If we inject the bundler as you suggested then NewTopic will no longer need the ctx arg.

Reply:
See #733.